apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/ExecutionContext.kt

```diff
@@ -4,6 +4,13 @@ import com.apollographql.apollo3.api.ExecutionContext.Element
 import com.apollographql.apollo3.api.ExecutionContext.Key
 import kotlin.jvm.JvmField
 
+@ApolloExperimental
+abstract class ClientContext(override val key: Key<*>): Element
+@ApolloExperimental
+abstract class RequestContext(override val key: Key<*>): Element
+@ApolloExperimental
+abstract class ResponseContext(override val key: Key<*>): Element
```

Comment:
should we provide docs?

Reply:
Yes, definitly docs 👍 . I'm not adding them first thing as the API changes **a lot** these days and tracking changes in the doc takes time. If that's ok with you, I'll make a `runtime` "API Grooming" PR like the [apollo-api one](https://github.com/apollographql/apollo-android/pull/2954) once things have settled a bit.